frontend/src/app/components/wp-fast-table/handlers/state/selection-transformer.ts

```diff
@@ -48,22 +46,9 @@ export class SelectionTransformer {
         this.renderSelectionState(state);
       });
 
-    // Bind CTRL+A to select all work packages
-    Mousetrap.bind(['command+a', 'ctrl+a'], (e) => {
-      this.wpTableSelection.selectAll(table.renderedRows);
 
-      e.preventDefault();
-      this.opContextMenu.close();
-      return false;
-    });
-
-    // Bind CTRL+D to deselect all work packages
-    Mousetrap.bind(['command+d', 'ctrl+d'], (e) => {
-      this.wpTableSelection.reset();
-      this.opContextMenu.close();
-      e.preventDefault();
-      return false;
-    });
+    this.wpTableSelection.registerSelectAllListener(() => { return table.renderedRows; });
+    this.wpTableSelection.registerDeselectAllListener();
```

Comment:
I think these are never unregistered (same for the old mousetrap binding). Maybe we should move them into the components with something like https://angular.io/guide/user-input#key-event-filtering-with-keyenter

```
    @HostListener('keydown.shift.tab', ['$event'])
```